src/ga4gh/vrs/extras/translator.py

```diff
@@ -438,11 +439,14 @@ def _from_hgvs(self, hgvs_dup_del_expr: str, **kwargs):
             copy_change: Copy change. If not provided, default is efo:0030067 for
                 deletions and efo:0030070 for duplications
         """
-        # sv = self._get_parsed_hgvs(hgvs_dup_del_expr)
         sv = self.hgvs_tools.parse(hgvs_dup_del_expr)
         if not sv:
             return None
 
+        if sv.type not in {"g", "m"}:
+            err_msg = "Only 'g' and 'm' reference sequences are supported"
+            raise ValueError(err_msg)
+
```

Comment:
@larrybabb was this comment for using the CnvTranslator or was it only for range positions?